.github/actions/e2e-prepare-containers/action.yml

```diff
@@ -66,14 +66,14 @@ runs:
 
         if ${{ inputs.postgres }}; then
           echo -e "${Y}Starting postgres container..." &&
-          docker run -d -p 5404:5432 metabase/qa-databases:postgres-sample-12 &&
+          docker run -d -p 5404:5432 -e TZ='US/Pacific' metabase/qa-databases:postgres-sample-12 &&
```

Comment:
what if we pass TZ as a global env variable instead? @metabase/core-ci

Reply:
I've tried to find a way how to do that reading [actions metadata syntax reference](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions), but I was unable to. According to that document, `runs`-global `env` may be specified only in `docker` action and we are using `composite`. Composite then can specify `env` per `runs.steps` entry, but I believe that is not desired. Could you please elaborate on how to achieve that? Thank you.

Reply:
you're right. I think it's fine to pass env vars right away, I checked docs and probably it will work if we pass env at the step level + pass this env to the docker, but it's a matter of taste and I'd not change anything in your version

👍